components/TextFields/src/ContainedInputView/MDCBaseTextField.h

```diff
@@ -105,4 +117,18 @@
  */
 - (nonnull UIColor *)textColorForState:(MDCTextControlState)state;
 
+/**
+ Sets the assistive label color for a given state.
+ @param assistiveLabelColor The UIColor for the given state.
+ @param state The MDCTextControlState.
+ */
+- (void)setAssistiveLabelColor:(nonnull UIColor *)assistiveLabelColor
```

Comment:
trying to understand: 
We are setting both leading and trailing colors here. It seems like a convenience method for more specific leading and trailing APIs. Why did we not create the setLeadingAssistiveLabelColor and setTrailingAssistiveLabelColor?

Reply:
I guess just because they always seem to be the same color in the guidelines. MDCTextField didn't allow you to set them separately either FWIW. Do you think we should have separate APIs for the leading and trailing assistive labels?

Reply:
I separated it into two accessors.